style.css

```diff
@@ -580,6 +590,69 @@ textarea {
   margin-left: 80px;
 }
 
+.popup_container {
+  z-index: 10000;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  background-color: rgba(124, 124, 124, 0.7);
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+}
+
+.popup {
+  background-color: white;
+  border-radius: 25px;
+  width: 75%;
+  padding: 1rem;
+}
+
+.popup_head {
+  display: flex;
+  justify-content: space-between;
+}
+
```

Comment:
- [ ] `required`
- [ ] as mentioned above in the desktop `pop-up`
- [ ] I suggest that you make some adjustments to the `pop-up cards`, please make them as much as similar to the figma template, kindly follow the background blur, fonts and how the background images are placed please see screenshots below

- `your template`
![Screenshot from 2022-07-21 20-11-34](https://user-images.githubusercontent.com/66230306/180284803-200f0a71-c2ad-4fb4-b7f3-2dfa8b5e8fd1.png)

- `figma template`
![Screenshot from 2022-07-21 19-59-38](https://user-images.githubusercontent.com/66230306/180284915-d6a717cc-4bae-4581-a490-980b14da8bdc.png)

Reply:
@mwanawabangona Thank you again.